docs/src/_guide/advanced-topics.md

```diff
@@ -13,3 +13,159 @@ Most of Meltano's features are available without installing any additional packa
+
+It is also possible to run [Meltano on GitHub Actions](https://github.com/brooklyn-data/meltano-on-github-actions). It’s likely possible to update this action to supporting docker-in-docker as well.
+
+There [are](zhsj.me/blog/view/dind-without-privileged) [several](shisho.dev/blog/posts/docker-in-docker) [articles](docs.docker.com/engine/security/rootless) on the web which discuss docker-in-docker in more detail.
+
+We are interested in exploring this challenge with the community. Please share in **this discussion or connect with us in Slack**about your experiences with attempting to put this into production.
+
+### How does this work with custom connectors made with the Airbyte CDK?
+
+This integration will work with any dockerized Airbyte source, whether it was made with their CDK or not.
+
+To configure your Airbyte connector as a [custom plugin]([https://docs.meltano.com/concepts/project#custom-plugin-definitions](https://docs.meltano.com/concepts/project#custom-plugin-definitions)) in Meltano you can copy the configuration shown below into your meltano.yml file then replace `name`, and the `value` for the airbyte_spec.image to reference your docker image. You can configure your connector without defining all of the settings and their metadata.
+
+```yaml
+ - name: tap-pokeapi # REPLACE THIS WITH YOUR CONNECTOR NAME
+   variant: airbyte
+   executable: tap-airbyte
+   namespace: tap_airbyte
+   pip_url: git+https://github.com/z3z1ma/tap-airbyte.git
+   capabilities:
+   - catalog
+   - state
+   - discover
+   - about
+   - stream-maps
+   - schema-flattening
+   settings:
+   - description: Airbyte image to run
+     kind: string
+     label: Airbyte Spec Image
+     name: airbyte_spec.image
+     value: airbyte/source-punk-api # REPLACE THIS WITH YOUR IMAGE NAME
+   - description: Airbyte image tag
+     kind: string
+     label: Airbyte Spec Tag
+     name: airbyte_spec.tag
+     value: latest
+   - [INSERT OTHER SETTINGS HERE]
+```
+
+### What features does this add to Airbyte connectors?
+
+Running Airbyte sources with Meltano brings a number of benefits to those connectors. With Meltano you get the ability to run stream maps to adjust data on the fly, you can define multiple environments to override configuration depending on where the EL pipeline is run, and you get the benefits of version control since everything is defined in your meltano.yml file.
+
+We also recently added support for alternative [state backends](https://docs.meltano.com/concepts/state_backends) to Meltano. This means if you want to store the incremental state between runs in a place other than your system database or local filesystem, you can! Currently Meltano supports AWS S3, Azure Blob Storage, and Google Cloud Storage as alternative state backends.
+
+Since tap-airbyte-wrapper was written with the SDK, this also unlocks the `BATCH` message format which can help with overall pipeline throughput to compatible targets.
```

Comment:
Has `BATCH` been tested with this? I'd be elated if it works out-of-the-box 😅

Reply:
@edgarrmondragon it does! It's not guaranteed to necessarily be faster since the tap still serializes it, but it does work!